python/paddle/fluid/tests/unittests/primitive/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+file(
```

Comment:
change dir name as prim, so that we can use same naming system

Reply:
done